client/blocks/login/login-form.jsx

```diff
@@ -257,6 +260,163 @@ export class LoginForm extends Component {
 		}
 	}
 
+	handleWooCommerceSubmit = event => {
+		event.preventDefault();
+		document.activeElement.blur();
+		if ( ! this.props.hasAccountTypeLoaded ) {
+			this.props.getAuthAccountType( this.state.usernameOrEmail );
+			return;
+		}
+		this.loginUser();
+	};
+
+	renderWooCommerce() {
+		const isFormDisabled = this.state.isFormDisabledWhileLoading || this.props.isFormDisabled;
+		const { requestError, socialAccountIsLinking: linkingSocialUser } = this.props;
+
+		return (
+			<form method="post">
+				{ this.renderPrivateSiteNotice() }
+
+				<Card className="login__form">
+					{ this.renderPrivateSiteNotice() }
```

Comment:
Does this need to be included here and in line 279 also?

Reply:
Thanks for catching that, fixed!